### PR TITLE
changed push link on clicking view content for harnoor and ml course

### PR DESCRIPTION
**Pull request by Abhijit-Jha**
### PR Fixes:
- 1 changed  redirected link from https://harkirat.classx.co.in/  to https://harkirat.classx.co.in/new-courses/8/content?activeTab=Content

Resolves #[318]

### CODE REVIEW

File: src/components/Courses.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const Courses = ({ courses }: { courses: Course[] }) => {
                 course.title.includes('Machine Learning') ||
                 course.title.includes('Harnoor')
               ) {
-                router.push('https://harkirat.classx.co.in/');
+                router.push(`https://harkirat.classx.co.in/new-courses/8/content?activeTab=Content`);
```
Comment (t007rushi) [inline src/components/Courses.tsx:21]: @Abhijit-Jha should we ask @hkirat to move this to projects.100xdevs cms page

Comment (t007rushi) [inline src/components/Courses.tsx:21]: @Abhijit-Jha should we ask @hkirat to move this to projects.100xdevs cms page

Comment (devsargam) [inline src/components/Courses.tsx:21]: How did u assume everyone has purchased 0-100?

**Comment by hkirat** [comment]
Same as what @devsargam  said, people have bought different courses

**Pull request closed without merging**